### DISCUSSION TITLE
Connect from a local laptop to a remote cluster

### DISCUSSION THREAD

**Discussion by YarShev** [dask/distributed#5455] [Q&A] [answered]
Hi guys, is there a way (API) to connect from a local laptop to a remote cluster to execute functions in parallel? Like in Ray - https://docs.ray.io/en/latest/cluster/ray-client.html

**Comment by YarShev** [comment, marked as answer]
Will the following work?

```python
# this notebook is in a local laptop
from distributed.client import Client
c = Client(address="node_ip_of_remote_cluster")
```

**Reply by andersy005** [reply]
As far as I know, this may work. However, it depends on whether the IP address of the scheduler is reachable from the laptop. So, if your scheduler is on private network (that can't be reached directly from your laptop), you may have some other issues to resolve.
